status-bar/ios/Sources/StatusBarPlugin/StatusBar.swift

```diff
@@ -0,0 +1,167 @@
+import Foundation
+import Capacitor
+
+@objc public class StatusBar: NSObject {
```

Comment:
I agree with Mark, you don't really need `@objc` or `NSObject`.

Nor would you need `super.init()`.